pkg/client_data/lib/package_api.dart

```diff
@@ -88,6 +88,24 @@ class Message {
   Map<String, dynamic> toJson() => _$MessageToJson(this);
 }
 
+@JsonSerializable()
+class PackageData {
+  final String name;
+  final VersionInfo latest;
```

Comment:
Can we add a note that this is merely a convenience property, because the `VersionInfo` for the latest version also exists in the `versions` list.

Reply:
Can we document that this is the latest that is NOT a pre-release, unless there only is pre-releases.